### PR TITLE
Add workaround for React Native border radius property due to issues …

### PR DESCRIPTION
**Pull request by jacobp100**
…with Image elements only accepting a single value for all radii

### CODE REVIEW

File: src/models/InlineStyle.js [hunk 1]
```diff
@@ -26,15 +26,20 @@ export default class InlineStyle {
     if (!generated[hash]) {
       const root = parse(flatCSS)
       const declPairs = []
+      const workaroundValues = {}
       root.each(node => {
         if (node.type === 'decl') {
-          declPairs.push([node.prop, node.value])
+          if (/border-?radius/i.test(node.prop)) {
```
Comment (mxstbr) [inline src/models/InlineStyle.js]: Let's add a comment here saying why we need this, linking to the relevant discussion.

**Comment by brunolemos** [comment]
Wasn't published to npm yet, right? Still having the issue

**Comment by jacobp100** [comment]
Sorry, it needs to be done. I'll try and get it done later or tomorrow!

**Comment by brunolemos** [comment]
@jacobp100 no problem

**Comment by mxstbr** [comment]
Ah sorry, do we need to publish a new `styled-components` version too?

**Comment by brunolemos** [comment]
I think so, there are changes in this repo

**Comment by mxstbr** [comment]
Okay will do today, sorry for the delay!

**Comment by brunolemos** [comment]
Just to inform, I'm still facing this issue on v1.1.3. cc @jacobp100

**Comment by jacobp100** [comment]
Could you `npm install css-to-react-native@latest` and see if that fixes it?

**Comment by brunolemos** [comment]
@jacobp100 it doesn't

**Comment by jacobp100** [comment]
Could you post the broken code?

**Comment by brunolemos** [comment]
@jacobp100 I'm not being able to reproduce in a new project 😫
So it's probably something with my project, even though I'm using the same versions and already cleaned the cache... I'll keep investigating.

**Comment by brunolemos** [comment]
@jacobp100 it's working fine after more cache cleans. 😅

**Comment by mxstbr** [comment]
Great! 🎉

**Pull request merged**